docs/source/tts/configs.rst

```diff
@@ -96,7 +96,7 @@ Text normalization (TN) converts text from written form into its verbalized form
       _target_: nemo_text_processing.text_normalization.normalize.Normalizer
       lang: en
       input_case: cased
-      whitelist: "nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv"
+      whitelist: "scripts/tts_dataset_files/en/whitelist_lj_speech.tsv"
```

Comment:
we don't have such a directory: `scripts/tts_dataset_files/en` due to legacy, instead, we have `scripts/tts_dataset_files` for English. Do you mind if creating an `en` sub-dir and move all resources inside it? This should align with other locales, like `zh` and `de`.

Reply:
Same changes should be applied all yaml configs that refer to these resources.

Reply:
Brief side note: we'll have to update some configs and tutorials to point to the right new places once this is done.

Reply:
ok, but why don't we update configs and tutorials in this PR? otherwise, all TTS recipes would collapse until a second PR with updated configs?

Reply:
discussed offline. removed this argument.